front/app.js

```diff
@@ -27,6 +28,7 @@ class App {
             alogliaSearch,
             confirmModal,
             emModal,
+            checkWithShift,
```

Comment:
this line activates the listener for all pages of the site, as it is not used could you remove it please?